### PR TITLE
es6-const-let

### PR DESCRIPTION
**Pull request by easton-morris**
image

### CODE REVIEW

File: es6-const-let/index.html [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    console.log('Random greeting:', getRandomGreeting(greetings));
+
+    function addToGreetings(newGreeting, greetingList) {
+      greetingList.push(newGreeting);
+    }
+    addToGreetings('namaste', greetings);
+    logAllGreetings(greetings);
+
+    console.log('This variable is currently undefined due to hoisting, please fix: ', variable);
+    const variable = 'meow';
```
Comment (BrettAlbright) [inline es6-const-let/index.html:45]: Make sure that you move this `const` above the console log to fix the undefined issue.

**Review by BrettAlbright — CHANGES_REQUESTED** [review]
small change noted

File: es6-const-let/index.html (hunk 1 above)
Comment (easton-morris) [reply es6-const-let/index.html:45]: Ah, I see. I thought we weren't supposed to move the declarations so I couldn't figure it out and just gave up lol

**Comment by easton-morris** [comment]
image

**Review by BrettAlbright — APPROVED** [review]
👍🏼

**Pull request merged**